src/compas/robots/model/base.py

```diff
@@ -90,7 +90,8 @@ def from_urdf(cls, attributes, elements=None, text=None):
         Examples
         --------
         >>> attributes = {'rpy': '0.0 1.57 0.0', 'xyz': '0.0 0.13 0.0'}
-        >>> FrameProxy.from_urdf(attributes, [], None)
+        >>> f = FrameProxy.from_urdf(attributes, [], None)
+        >>> print(f)
```

Comment:
this can't be handled with forwarding the `__repr__`?

Reply:
yep, you're right. fixed